packer/resources/features/mongo-opsmanager/scripts/snapshot_backup.rb

```diff
@@ -92,15 +95,15 @@ def download_import_team_keys
   logger.info("Downloading document #{keyring_document_key} from bucket #{@options.keys_bucket}")
   s3 = Aws::S3::Client.new
   s3.get_object({bucket: @options.keys_bucket, key: keyring_document_key}, target: '/tmp/flexbackupkeys.gpg')
-  `gpg --homedir /home/mongo-backup/ --import /tmp/flexbackupkeys.gpg`
+  `gpg --homedir #{GPG_HOME_DIR} --import /tmp/flexbackupkeys.gpg`
 end
 
 def generate_gpg_command(filename)
-  keys = `gpg --homedir /home/mongo-backup/ --list-keys | grep uid`
-  key_uid_list = keys.split('\n')
```

Comment:
This is a very important change! String interpolation is needed for newline characters - http://stackoverflow.com/questions/16406381/ruby-split-n-not-splitting-on-new-line